client/js/main.js

```diff
@@ -5,14 +5,29 @@ import store from './store/store';
 import router from './router';
 Vue.config.productionTip = false;
 
-const app = new Vue({
-  vuetify,
-  store,
-  router,
-  render: (h) => h(App),
-}).$mount('#app');
+async function initApp() {
+  // Make sure we try and reestablish a player's session if one exists
+  // We do this before the app mounts to preempt the reauth/logout logic
+  try {
+    await store.dispatch('requestStatus');
+  } catch {
+    // Unable to validate player status or player not authenticated
+    // Swallow the error to let the player continue
+  }
```

Comment:
This will set the user's auth status so if you reload on the homepage, the stats page or the game list page it'll keep you logged in.